scripts/conflicts.py

```diff
@@ -80,7 +80,7 @@ def update_comment(dry_run, pull, pulls_conflict):
     text += "Reviewers, this pull request conflicts with the following ones:\n"
     text += "".join(
         [
-            f"\n* [#{p.CON_id.removeprefix(pull.CON_slug)}]({p.html_url}) ({p.title.strip()} by {p.user.login})"
+            f"\n* #{p.number} by {p.user.login}"
```

Comment:
The number isn't sufficient to identify a pull request in several monotree repos.

Also, this will remove the title in email notifications?

Reply:
> The number isn't sufficient to identify a pull request in several monotree repos.

Ah, I hadn't thought of that. Providing the full PR URL would fix that issue:
![image](https://user-images.githubusercontent.com/22493292/198823787-eb0218fe-a3b3-4ac9-9ca7-39251148a62d.png)

> Also, this will remove the title in email notifications?

I don't have a solution for this, feel free to close this PR if you feel this is important.

Reply:
What's great with this change is the merge status of the PR and maybe the title that gets live-updated.
Also a bit more readable.